contrib/app/sofast/run_fixed_pattern_projection.py

```diff
@@ -1,38 +1,33 @@
-"""Script that displays a fixed pattern on a projector-screen system for 1 second
-
-NOTE: This example requires a computer screen
+"""Script that displays a fixed pattern on a projector-screen system. Press the "escape"
+key to close the window.
 """
 
 from os.path import join
 
-import pytest
-
 from opencsp.app.sofast.lib.SystemSofastFixed import SystemSofastFixed
 from opencsp.common.lib.opencsp_path.opencsp_root_path import opencsp_code_dir
 from opencsp.common.lib.deflectometry.ImageProjection import ImageProjection
 
 
-@pytest.mark.no_xvfb
-def example_project_fixed_pattern():
+def run_fixed_pattern_projection():
     """Projects fixed pattern image on display"""
-    # Set pattern parameters
-    file_image_projection = join(
-        opencsp_code_dir(), "test/data/measurements_sofast_fringe/general/Image_Projection_test.h5"
-    )
-    width_pattern = 3
-    spacing_pattern = 6
 
     # Load ImageProjection
+    file_image_projection = join(
+        opencsp_code_dir(), "../../sofast_calibration_files/Image_Projection_optics_lab_landscape_square.h5"
```

Comment:
Same comment here; should this directory be passed in as a parameter?

Reply:
This now is consistent with run_and_characterize_sofast_fixed.py, but if you'd prefer all the inputs to be moved into the 
'''
if __name__ == '__main__':
    # Start processing
'''
block, I'd be happy to move them.

Reply:
That would make it easier to adapt this script to another system, and so I think we should follow this pattern whenever possible. If you wouldn't mind doing that it'd be great. Or, move them later in another ticket at some point.